src/styles/main.scss

```diff
@@ -297,6 +297,12 @@ ol {
 }
 
 
+svg {
```

Comment:
This removes the icon SVG from the document flow, so `document.body.offsetHeight` correctly reflects the height of the results.